torch/amp/autocast_mode.py

```diff
@@ -170,7 +170,8 @@ def forward(self, x):
     (see :ref:`Working with Multiple GPUs<amp-multigpu>`).
 
     Args:
-        device_type(str, required):  Whether to use 'cuda' or 'cpu' device
+        device_type(str, required):  Whether to use 'cuda' or 'cpu' device.
+                                     The type can be obtained by `device.type`.
```

Comment:
I think `device.type` would need more explanation as it's unclear to me where this object should come from.

Reply:
It's like `some_tensor.device.type`.